src/components/JsonCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from "react";
+import {Box, useTheme} from "@mui/material";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+
+const TEXT_COLOR_LIGHT = "#0EA5E9";
+const TEXT_COLOR_DARK = "#83CCED";
+const BACKGROUND_COLOR = "rgba(14,165,233,0.1)";
+
+type JsonCardProps = {
+  data: any;
```

Comment:
should we use a specific type here?

Reply:
It's not quite possible because this component will be reused by `event.data` whose type is `any`.